app/models/groups_metadata_board.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class GroupsMetadataBoard < ApplicationRecord
```

Comment:
It looks weird that it has only one row (not only for board, but also future GroupsMetadataCouncils). But reusing teams/committees metadata looks more weird for me because there are extra metadata for teams_committees like status (for now maybe only one). But both board & teams_committees metadatas can expand but in different directions - like for Board, we might have metadatas like term_duration, absence_count, term_doc_link, etc and teams_committees might have metadatas like mentor, days_since_promotion, seniority_points, etc. (I agree all these metadatas are unrealistic, but trying to explain that it can go in different directions if we ever decide to expand the metadatas).

So is it fine if we have the metadata? (another option I can think of is hard-code the board's email ID in the code as it is never going to change. But it will be tough to maintain when the user_roles & user_groups are used across different micro-services)

Reply:
Sure, that explanation makes total sense! Thank you :)